fearless/Modules/StakingPool/StakingPoolMain/StakingPoolMainProtocols.swift

```diff
@@ -54,6 +55,8 @@ protocol StakingPoolMainInteractorOutput: AnyObject {
     func didReceive(palletIdResult: Result<Data, Error>)
     func didReceive(poolAccountInfo: AccountInfo?)
     func didReceive(existentialDepositResult: Result<BigUInt, Error>)
+    func didReceive(nomination: Nomination?)
+    func didReceive(nominationError: Error)
```

Comment:
you can make one methods for delivery error in presenter. If you want known place where error throws you can make enum with error places